linkingExamples_test.go

```diff
@@ -0,0 +1,133 @@
+package ipld_test
+
+import (
+	"fmt"
+
+	"github.com/ipfs/go-cid"
+
+	"github.com/ipld/go-ipld-prime"
+	_ "github.com/ipld/go-ipld-prime/codec/dagcbor"
+	"github.com/ipld/go-ipld-prime/fluent"
+	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
+	basicnode "github.com/ipld/go-ipld-prime/node/basic"
+	"github.com/ipld/go-ipld-prime/storage"
+)
+
+// storage is a map where we'll store serialized IPLD data.
+//
+// ExampleCreatingLink will put data into this;
+// ExampleLoadingLink will read out from it.
+//
+// In a real program, you'll probably make functions to load and store from disk,
+// or some network storage, or... whatever you want, really :)
+var store = storage.Memory{}
```

Comment:
In fact, I think both examples need to be a single one; otherwise, running only the Load one will fail. This will be a problem for many scenarios like pkg.go.dev and godoc.org, where you can click "Run" on a single runnable example.

Reply:
Oh, boy.  Yeah.  I ported this example somewhat uncritically.  But you're right, it was wrong both now and before; that standalone property is really important.  I'll separate these.

...although I'm not really sure how.  Is there a good idiom for using the actions of one example as setup for another?  If I extract a function, it disappears from the rendered example in godoc again.  But the `Example*` function contract doesn't allow enough parameters to use it directly, either.

Reply:
Maybe I wasn't clear - I mean having a single Example function, and putting all the code in there. You can't have two separate example funcs that share state. It would be a bit like having two Test funcs that share global state, but worse, since running a single Example func is their main use case for the web docs.